portfolio/src/main/webapp/script.js

```diff
@@ -35,26 +35,32 @@ function addRandomFact() {
 /**
 * Retrieves comments when page loads.
 */
-const getComments = () => {  
+const getComments = () => {
   const numberOfComments = document.getElementById('numberOfComments').value;
   const typeOfComments = document.getElementById('typeOfComments').value;
   fetch(`/list-comments?numberOfComments=${numberOfComments}&orderBy=${typeOfComments}`
-    ).then(response => response.json()).then((comments) => {
-    const commentContainerElement = document.getElementById('comment-container');
-    const titleElement = document.createElement('h2');
-
-    // Clean element
-    commentContainerElement.innerHTML = '';
-
-    // If no comments, display an alert, else append comments to the container element.
-    if (comments.length === 0) {
-      titleElement.innerHTML = "There are no comments 😕, be the first!";
-      commentContainerElement.appendChild(titleElement);
-    }
-    else {
-      titleElement.innerHTML = "Last comments";
-      commentContainerElement.appendChild(titleElement);
-      appendComments(comments, commentContainerElement);
+  ).then(response => {
+    if (response.redirected) {
+      window.location.href = response.url;
+    } else {
+      response.json().then((comments) => {
```

Comment:
Nit: It's good to avoid nested if loops when it's possible. What do you think about the following:

```suggestion
    if (response.redirected) {
      window.location.href = response.url;
      return; // You can use this technique to avoid using "else" statements.
    }
    response.json().then((comments) => {
```

Reply:
Seems good, I was not aware about avoiding nested if statements. I'll update that.